HabitRPG/UI/RewardViewController.swift

```diff
@@ -134,6 +152,24 @@ class RewardViewController: BaseCollectionViewController, UICollectionViewDelega
             editedReward = nil
         }
     }
+    
+    private func createCollectionViewLayout() -> UICollectionViewLayout {
+        let layout = UICollectionViewCompositionalLayout { [self] sectionIndex, _ in
+            let isCustomRewardsSection = self.dataSource.isCustomRewardsSection(sectionIndex)
+            let sectionType: RewardSectionType = isCustomRewardsSection ? .custom : .inApp
+            let itemSize = NSCollectionLayoutSize(widthDimension: sectionType.itemWidth,
+                                                  heightDimension: sectionType.itemHeight)
+            let item = NSCollectionLayoutItem(layoutSize: itemSize)
+            let groupSize = NSCollectionLayoutSize(widthDimension: .fractionalWidth(1),
+                                                   heightDimension: sectionType.itemHeight)
+            let group = NSCollectionLayoutGroup.horizontal(layoutSize: groupSize, subitems: [item])
+            group.interItemSpacing = .flexible(1)
```

Comment:
This is the minimum spacing between items, we need this so `collectionView.autoresizingMask` will work properly and items will fill the width of the screen, without these, cells would be aligned to the left